package.json

```diff
@@ -30,6 +30,7 @@
     "@types/aws-lambda": "^8.10.77",
     "@types/bunyan": "^1.8.6",
     "@types/chai-as-promised": "^7.1.4",
+    "@types/debug": "^4.1.12",
```

Comment:
Is this related to the DEBUG env variable added above? Do we actually need to add this dependency?